modules/fb_instant_articles_display/src/InstantArticle.php

```diff
@@ -69,19 +68,20 @@ class DrupalInstantArticleDisplay {
    * @param $instantArticle
    */
   private function __construct($node, $layoutSettings, $instantArticle) {
+    parent::__construct();
```

Comment:
Either way.  Locally if I just remove this line, all is well and the members can remain private.  If we move to setting the vars in `create()`, then the members have to be public or private with setters.  You're call.